vertical-pod-autoscaler/pkg/updater/priority/scaling_direction_pod_eviction_admission_test.go

```diff
@@ -157,6 +157,26 @@ func TestAdmitForSingleContainer(t *testing.T) {
 		assert.Equal(t, true, sdpea.Admit(podWithoutRequests, recommendation))
 	})
 
+	t.Run("it should admit a Pod for eviction if it has non-nil ContainerStatus resources, but with no requests", func(t *testing.T) {
```

Comment:
should we maybe exercise this test a bit differently where ContainerStatus resources are large enough to trigger the eviction but requests are not?

That way, we can catch any (unlikely) regression where we start reading requests instead of container status.

Reply:
that's much better, thanks!